aBitMondrian.py

```diff
@@ -0,0 +1,32 @@
+import random
+
+def drawRect(x,y,width, height, color='black'):
+    return f'<rect x="{x}" y="{y}" width="{width}" height="{height}" fill="{color}" stroke-width="0.5" stroke="{color}"/>'
+
+def drawMondrian(filename,height, width, colors, colornums, step=10):
+    header = f'<svg viewBox="0 0 {WIDTH} {HEIGHT}" xmlns="http://www.w3.org/2000/svg">\n'
+    footer = f'</svg>'
+    sv = open(filename+".svg","w")
+    sv.write(header)
+
+    for y in range(0,height, step):
```

Comment:
> I think the diagonal lines code is the most complex here. I think you can simplify it a lot though.
> 
> General thoughts on running the code: I'd not open files. I'd instead just print to stdout (either `print` or `sys.stdout.write(...)`, and as a user, I can choose where to pipe the output. e.g.:
> 
> ```shell
> ./aBitMondrian.py > mondrian.svg
> ```
This is good advice but I'm actually implementing the code in repl.it to make svg as a webpage (cheating) so that it updates live. Pretty sure that i need to build a string for that. So the final version is a bit different from the one you can see. Actually, you're advice would make it more portable between the two so, um, yeah. I'll think about it